server/main.py

```diff
@@ -125,6 +120,10 @@ async def index(request):
 
     url = run_batch_job_and_print_url(batch, wait=params.get('wait', False))
 
+    # Publish the metadata to Pub/Sub. Wait for the result before running the batch.
```

Comment:
The comment is a bit misleading now :)

Reply:
Thanks for the catch! I've updated the comment :)